src/components/Common/Inputs/DatePicker.tsx

```diff
@@ -59,10 +59,11 @@ export function DatePicker<C extends FieldValues, N extends FieldPath<C>>({
   const { t } = useTranslation()
   const {
     field,
-    fieldState: { invalid },
+    fieldState: { invalid, error },
   } = useController({ name, control })
 
   const dateInputContainerRef = useRef<HTMLDivElement | null>(null)
+  const errorText = error?.message ? error.message : errorMessage
```

Comment:
I'm going to clean this up when we do the RHF separation, but noticed that datepicker wasn't displaying the right errors.